lib/api/jobRouter.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const router = express.Router();
+
+const jobController = require("../dbConnections/jobController");
+const { Status } = require("../constants/responseAndStatus");
+const logger = require("../logger/log");
+
+router.post("/create", (req, res) => {
```

Comment:
REST guidelines for the HTTP routes must be followed. Verbs should not be put in the routes, actions must be designated via HTTP methods